index.html

```diff
@@ -4,50 +4,59 @@
 		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
 		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
 		<link rel="stylesheet" href="./style.css" />
+		<link
+			rel="stylesheet"
+			href="https://use.fontawesome.com/releases/v5.13.1/css/all.css"
+		/>
 		<title>Document</title>
 	</head>
 	<body>
-		<div class="nav">
-			<div>menu button</div>
+		<div id="nav">
+			<div class="hamburgerBtn">
+				<div class="icon"><i class="fas fa-bars"></i></div>
+			</div>
 			<div class="menu">
-				<div class="icon">logo</div>
+				<div class="logo">
+					<div class="icon">logo</div>
+				</div>
 				<div class="iconsWrapper">
```

Comment:
please make use of BEM pattern (a new exersize was added, for the moment you can leave it)